From aad47cabc464af2cfda7dde1a5c2181f634859d6 Mon Sep 17 00:00:00 2001
Message-Id: <aad47cabc464af2cfda7dde1a5c2181f634859d6.1663622629.git.stefan@agner.ch>
From: Stefan Agner <stefan@agner.ch>
Date: Mon, 19 Sep 2022 23:05:51 +0200
Subject: [PATCH] Revert Python dependencies

Revert to dbus-python 1.2.16 and remove the crypto libraries. They are
not required in regular use.

This is available in Debian bullseye end seemed to have worked fine so
far.
---
 src/controller/python/BUILD.gn | 7 +------
 1 file changed, 1 insertion(+), 6 deletions(-)

diff --git a/src/controller/python/BUILD.gn b/src/controller/python/BUILD.gn
index 890c7fc850..5a66bec8eb 100644
--- a/src/controller/python/BUILD.gn
+++ b/src/controller/python/BUILD.gn
@@ -287,18 +287,13 @@ chip_python_wheel_action("chip-core") {
     "ipdb",
     "deprecation",
     "mobly",
-
-    # Crypto libraries for complex tests and internal Python controller usage
-    "cryptography",
-    "pycrypto",
-    "ecdsa",
   ]
 
   if (current_os == "mac") {
     py_package_reqs += [ "pyobjc-framework-corebluetooth" ]
   } else if (current_os == "linux") {
     py_package_reqs += [
-      "dbus-python==1.2.18",
+      "dbus-python==1.2.16",
       "pygobject",
     ]
   }
-- 
2.37.3

